app/routes.php

```diff
@@ -1,581 +0,0 @@
-<?php
```

Comment:
Not really part of this PR, more like fixing #1288 where I did not pay attention to the bad interaction between  git and macOS case-insensitive file system. Could you place this deletion at least in a separate commit?